examples/get-started/create-react-app/src/map.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react';
+import MapGL, { NavigationControl, FullscreenControl } from 'react-map-gl';
+import './map.css';
+
+// Insert your Mapbox API key here
+const API_KEY = '';
+
+class Map extends Component {
+  state = {
+    viewport: {
+      latitude: 37.785164,
+      longitude: -100,
+      zoom: 3.5,
+      bearing: 0,
+      pitch: 0
+    },
+    popupInfo: null
```

Comment:
This doesn't seem to be used

Reply:
True.  (A copy-paste error.)

Reply:
Good eye!  Sorry for the copy-paste error.